src/app/beer_garden/command_publishing_blocklist.py

```diff
@@ -0,0 +1,76 @@
+from brewtils.models import Operation
+
+import beer_garden.config as config
+from beer_garden.db.mongo.models import CommandPublishingBlockList
```

Comment:
The model name seems to be the last one standing in the great war over naming consistency. It's still `BlockList` camel case, rather than `Blocklist` like everything else.  It's not the end of the world honestly, but it should be an easy enough thing to change so I vote that we change it and then everything will be consistent.